src/main/kotlin/com/terraformation/backend/customer/api/ProjectsController.kt

```diff
@@ -122,6 +127,42 @@ class ProjectsController(
 
     return SimpleSuccessResponsePayload()
   }
+
+  @Operation(summary = "Get all internal project users for a project.")
+  @GetMapping("/{id}/userRoles")
```

Comment:
This could be `internalUsers` or `internalRoles` instead, just to make it super clear that it's not about end-user roles.

Reply:
Good thought, I like that. Updated